### DISCUSSION TITLE
Add Headset icon?

### DISCUSSION THREAD

**Discussion by KayleyHill** [lucide-icons/lucide#1314] [Show and tell]
![callcenter-headset](https://github.com/lucide-icons/lucide/assets/135126101/5bac86f0-13b0-40b8-9484-ee52f1147fbe)

Been using feather/lucide for our icons on our stuff and wanted to give back with a headset icon I created based off the headphones. How do I submit this for consideration? I am very new to github.

**Comment by karsa-mistmere** [comment]
Please submit either an [icon request issue](https://github.com/lucide-icons/lucide/issues/new?assignees=&labels=%F0%9F%99%8C+icon+request&projects=&template=icon_request.md) or a [pull request](https://github.com/lucide-icons/lucide/compare) containing the icon according to our [contribution guide](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md) and [icon design guide](https://lucide.dev/docs/icon-design-guide).
